scipy/sparse/linalg/eigen/arpack/arpack.py

```diff
@@ -1690,224 +1690,3 @@ def eigsh(A, k=6, M=None, sigma=None, which='LM', v0=None,
             params.iterate()
 
         return params.extract(return_eigenvectors)
-
-
-def _augmented_orthonormal_cols(x, k):
-    # extract the shape of the x array
-    n, m = x.shape
-    # create the expanded array and copy x into it
-    y = np.empty((n, m+k), dtype=x.dtype)
-    y[:, :m] = x
-    # do some modified gram schmidt to add k random orthonormal vectors
-    for i in range(k):
-        # sample a random initial vector
-        v = np.random.randn(n)
-        if np.iscomplexobj(x):
-            v = v + 1j*np.random.randn(n)
-        # subtract projections onto the existing unit length vectors
-        for j in range(m+i):
-            u = y[:, j]
-            v -= (np.dot(v, u.conj()) / np.dot(u, u.conj())) * u
-        # normalize v
-        v /= np.sqrt(np.dot(v, v.conj()))
-        # add v into the output array
-        y[:, m+i] = v
-    # return the expanded array
-    return y
-
-
-def _augmented_orthonormal_rows(x, k):
-    return _augmented_orthonormal_cols(x.T, k).T
-
-
-def _herm(x):
-    return x.T.conj()
-
-
-def svds(A, k=6, ncv=None, tol=0, which='LM', v0=None,
-         maxiter=None, return_singular_vectors=True,
-         solver='arpack'):
-    """Compute the largest or smallest k singular values/vectors for a sparse matrix. The order of the singular values is not guaranteed.
-
-    Parameters
-    ----------
-    A : {sparse matrix, LinearOperator}
-        Array to compute the SVD on, of shape (M, N)
-    k : int, optional
-        Number of singular values and vectors to compute.
-        Must be 1 <= k < min(A.shape).
-    ncv : int, optional
-        The number of Lanczos vectors generated
-        ncv must be greater than k+1 and smaller than n;
-        it is recommended that ncv > 2*k
-        Default: ``min(n, max(2*k + 1, 20))``
-    tol : float, optional
-        Tolerance for singular values. Zero (default) means machine precision.
-    which : str, ['LM' | 'SM'], optional
-        Which `k` singular values to find:
-
-            - 'LM' : largest singular values
-            - 'SM' : smallest singular values
-
-        .. versionadded:: 0.12.0
-    v0 : ndarray, optional
-        Starting vector for iteration, of length min(A.shape). Should be an
-        (approximate) left singular vector if N > M and a right singular
-        vector otherwise.
-        Default: random
-
-        .. versionadded:: 0.12.0
```

Comment:
These `versionadded` tags appear to missing in the updated docstring file.  Was this intentional?  I don't we need to propagate these forever, just curious

Reply:
Let's say it was not unintentional. I don't think they need to be purged, but as I was working on this I didn't feel like they needed to preserved forever.